app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class CardsMailer < ActionMailer::Base
+  layout 'mailer'
+  default from: ENV['CARDS_MAILER_DEFAULT_FROM']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.